dl/src/main/scala/com/intel/analytics/bigdl/utils/DLClassifier.scala

```diff
@@ -52,7 +52,7 @@ class DLClassifier[@specialized(Float, Double) T: ClassTag]
    */
   def setOutputCol(outputColName: String): this.type = set(outputCol, outputColName)
 
-  override def validateParams(): Unit = {
+  def validateParameters(): Unit = {
```

Comment:
In spark mllib 2.1, trait Params does not have validateParams() any more. Or we may keep the DLClassifier code, and create Params wrapper in spark-version?